src/GeoExt/tree/Util.js

```diff
@@ -0,0 +1,56 @@
+Ext.define('GeoExt.tree.Util', {
+    statics: {
+        /**
+         * Updates the visibility of the layer that is connected to given
+         * node.
+         *
+         * @private
```

Comment:
why is one method marked private and the other one not, when they are both used externally?